### PR TITLE
Fix Hive 3.x  kerberos not Support

### PR DESCRIPTION
**Pull request by flaming-archer**
### :pencil: Description
Have errors that  some zookeeper class not found.

### CODE REVIEW

**Comment by flaming-archer** [comment]
I cherry pick https://github.com/ExpediaGroup/waggle-dance/pull/266, I found some change not merge to hive3.x branch.

File: waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/FederatedMetaStoreStorage.java [hunk 1]
```diff
@@ -31,4 +31,5 @@ public interface FederatedMetaStoreStorage {
 
   void update(AbstractMetaStore oldMetaStore, AbstractMetaStore newMetaStore);
 
+  void saveFederation();
```
Comment (patduin) [inline waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/FederatedMetaStoreStorage.java:34]: can you update your branch it looks like the PR contains changes you already merged.

Comment (flaming-archer) [reply waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/FederatedMetaStoreStorage.java:34]: Sorry,I will fix it later.

Comment (flaming-archer) [reply waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/FederatedMetaStoreStorage.java]: > can you update your branch it looks like the PR contains changes you already merged.

Done.

**Review by patduin — APPROVED** [review]

**Review by abhimanyugupta07 — APPROVED** [review]

**Pull request merged**